test/integration/targets/mongodb_replicaset/files/js/is_primary.js

```diff
@@ -0,0 +1,13 @@
+var done = false;
+var iterations = 0;
+while(rs.status()['myState'] != 1) {
+		if (!done) {
+			//print("State is not yet PRIMARY. Waiting...");
+			done = true
+		}
+		sleep(10000);
```

Comment:
```suggestion
		sleep(1000);
```

Reply:
I've accepted this but MongoDB typically takes 10-30 seconds to declare a primary.